index.js

```diff
@@ -39,20 +39,34 @@ client.settings = new PersistentCollection({name: "settings"});
 
 const init = async () => {
 
-  // Here we load **commands** into memory, as a collection, so they're accessible
-  // here and everywhere else.
-  const cmdFiles = await readdir("./commands/");
-  client.log("log", `Loading a total of ${cmdFiles.length} commands.`);
-  cmdFiles.forEach(f => {
+  /*
+  Here we load **commands** into memory, as a collection,
+  so they're accessible here and everywhere else.
+  Useing the pathwalker function out of functions.js.
+  */
+  const cmdFiles = await client.pathwalker("./commands/");
```

Comment:
Why add a function to the client if you're only going to use it once? You should only make functions that you're going to use multiple times.